src/modules/textFacets/__tests__/textFacets.test.js

```diff
@@ -402,5 +404,52 @@ test('Test selected facet click on text Facet with FacetItemComponent', async ()
     })
 });
 
+test('Test view more on text facet', async () => {
+    const { getByText, container } = render(
+        <>
+            <UnbxdSearchWrapper
+                siteKey="wildearthclone-neto-com-au808941566310465"
+                apiKey="e6959ae0b643d51b565dc3e01bf41ec1"
+            >   
+                <TextFacets enableViewMore={true} minViewMore={3}/>
+                <Products
+                    attributesMap={attributesMap}
+                />
+                <div>
+                    <SearchBox defaultSearch="shoes" />
+                </div>
+            </UnbxdSearchWrapper>
+        </>
+    );
+
+    await waitFor(async () => {
+        fireEvent.click(container.getElementsByClassName('view-More')[0]);
```

Comment:
When you click on view more, you must see view less, Right?

Reply:
just the class-name is view-more but initially the list is expanded and view less is shown for all facets